ws.ts

```diff
@@ -64,16 +75,54 @@ async (req: Request, info: Deno.ServeHandlerInfo) => {
                 socket.close(3000, rawEvent.message);
                 return;
             }
-            const event = parseJSON<NostrEvent>(rawEvent);
+            const event = parseJSON<NostrEvent | Event_V2>(rawEvent);
             if (event instanceof Error) {
                 console.error(event);
                 socket.close(3000, "invalid auth event format");
                 return;
             }
-
-            const ok = await args.is_member(event.pubkey);
-            if (!ok) {
-                socket.close(3000, `pubkey ${event.pubkey} is not allowed`);
+            if (event.kind == NostrKind.HTTP_AUTH) {
+                const ok = await args.is_member(event.pubkey);
+                if (!ok) {
+                    socket.close(3000, `pubkey ${event.pubkey} is not allowed`);
+                    return;
+                }
+            } else if (event.kind == Kind_V2.AcceptInvitation) {
```

Comment:
Let's not mix Invitation with WebSocket

Reply:
Assuming I creates an invitation for `deno.blowater.app`

The user flow should be clicking an url such as `blowater.app/the-invite-event-base64` which makes it a HTTP GET request.

This GET REQUEST is to the blowater app itselft, not a relay.

Then, the app decodes the event from the URL and then does a
HTTP POST with the same event to the the relay `deno.blowater.app`.

Then the rest follows.

Reply:
Yes, your solution is better. 

Because the role of Accept Event is to add pubkey to the allow list, which can be completely decoupled from auth. 

So POST request can be used entirely. Then establish a websocket connection later on.